src/App.js

```diff
@@ -8,6 +8,7 @@ import { connect } from 'react-redux'
 import { fetch_advising } from './actions/advisingActions'
 import { fetch_terms } from './actions/termsActions'
 import { withStyles } from '@material-ui/core/styles'
+// import { makeStyles } from '@material-ui/styles'
```

Comment:
We can probably just delete that commented out guy

```suggestion
```

Reply:
Hmm, yea I definitely should have refactored before pushing but I figured that would be better of in another PR I'm working on as it moves over to using functions.